airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresJdbcSourceAcceptanceTest.java

```diff
@@ -234,14 +232,6 @@ public boolean supportsSchemas() {
     return true;
   }
 
-  @Test
-  void testSpec() throws Exception {
```

Comment:
why removed?

Reply:
There's a function in the parent class which is pretty much doing the same exact thing : https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/kotlin/io/airbyte/cdk/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.kt#L193